spa/src/pages/ConfigSteps/empty-state-flow.test.tsx

```diff
@@ -0,0 +1,141 @@
+import "@testing-library/jest-dom";
+import { act, render, screen, fireEvent, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ConfigSteps from "./index";
+import Connected from "../Connected";
+import { MemoryRouter, Route, Routes } from 'react-router-dom'
+import * as nock from "nock";
+
+describe("Empty state user flow", () => {
+
+	let mockWinOpenFunc: jest.Mock;
```

Comment:
/nit As someone looking at this with no context, the function name isn't super clear. Does it mean `mockWindowOpenFunc`?

Reply:
I've abstracted this away.

Reply:
have abstracted it away